### ISSUE TITLE
Doesn't build anymore on NixOS unstable.

### ISSUE THREAD

**Issue by magnetophon** [zynaddsubfx/zyn-fusion-issues#213]
I get:
```
[ 70%] Building CXX object src/CMakeFiles/zynaddsubfx_core.dir/Misc/Recorder.cpp.o
/build/zynaddsubfx-3.0.3/src/Misc/XMLwrapper.cpp: In function 'const char* zyn::XMLwrapper_whitespace_callback(mxml_node_t*, int)':
/build/zynaddsubfx-3.0.3/src/Misc/XMLwrapper.cpp:37:28: error: invalid use of incomplete type 'mxml_node_t {aka struct _mxml_node_s}'
     const char *name = node->value.element.name;
                            ^~
In file included from /build/zynaddsubfx-3.0.3/src/Misc/XMLwrapper.h:16:0,
                 from /build/zynaddsubfx-3.0.3/src/Misc/XMLwrapper.cpp:16:
/nix/store/j8rm6k32q3p00kr3k009i2b8vvnxi7jk-mxml-3.0/include/mxml.h:97:16: note: forward declaration of 'mxml_node_t {aka struct _mxml_node_s}'
 typedef struct _mxml_node_s mxml_node_t; /**** An XML node. ****/
                ^~~~~~~~~~~~
/build/zynaddsubfx-3.0.3/src/Misc/XMLwrapper.cpp: In member function 'void zyn::XMLwrapper::endbranch()':
/build/zynaddsubfx-3.0.3/src/Misc/XMLwrapper.cpp:294:53: error: invalid use of incomplete type 'mxml_node_t {aka struct _mxml_node_s}'
         cout << "endbranch()" << node << "-" << node->value.element.name
```
and lots of similar lines.

**Comment by fundamental** [comment]
mxml 3.0 compatibility is already available in the 3.0.4 release.

**Comment by magnetophon** [comment]
Thanks, that indeed builds just fine.
